operator/deploy/deploy-kiali-operator.sh

```diff
@@ -143,6 +143,10 @@
 #    already (or will) contain the credentials (i.e. the secret you must create manually).
 #    Default: kiali
 #
+# SYSTEM_TEST_CODECOV_ENABLED
```

Comment:
i don't like this change here. This file user-facing and is for users to install the operator. Injecting things like this for only internal CI usage seems wrong.

We should introduce a new .sh file for internal CI testing use only - and have it call this file if need be.